header_test.go

```diff
@@ -294,3 +294,70 @@ func TestFormat(t *testing.T) {
 		}
 	}
 }
+
+func TestHeaderProxyFromAddrs(t *testing.T) {
```

Comment:
Would be great not to reduce coverage but I can survive it. Just letting you know.